package/suse-migration-services-spec-template

```diff
@@ -132,6 +135,8 @@ install -D -m 644 systemd/suse-migration-console-log.service \
 %{_bindir}/suse-migration-mount-system
 %{_unitdir}/suse-migration-mount-system.service
 
+%{_unitdir}/suse-migration-pre-checks.service
```

Comment:
suse-migration-run-pre-checks.service?

Reply:
Are you asking about the typo I had (thanks for finding),  I just fixed the typo or the service. .  If you are asking about the services this PR will add a "pre-checks"  systemd service that runs suse-migration-pre-checks when the system boots from the migration iso.  It checks if the original system (mounted as /system-root) passed the pre-checks. if it detects the multi kernel issue, it will remediate before starting the upgrade/

Reply:
I was asking about the typo. Looks good now.